guides/release/components/template-tag-format.md

```diff
@@ -0,0 +1,285 @@
+The template tag format is a powerful, new way to write components in Ember. It's a single-file format that combines the component's JavaScript and Glimmer template code. The `<template>` tag is used to keep a clear separation between the template language and the JavaScript around it.
+
+Template tag components use the file extension `.gjs`. This abbreviation is short for "Glimmer JavaScript". The file extension `.gts` is also supported for TypeScript components.
+
+This new format is the official future of Ember's component authoring story, and is stable and usable today. We expect it to become the recommended and default way of authoring all Ember apps in the near future, once we are satisfied that we have sufficiently polished up all the corners of the implementation.
```

Comment:
Since we're talking about this being "official" it might be worth linking to the RFC here and mentioning what stage it's in? thoughts?

Reply:
I've included a link to the RFC. Repeating the stage here makes me concerned it might get forgotten when it moves to a next stage.

Reply:
we did think about that, but the next stage is "ready for release" and the one after that is "recommended". To move to "recommended" we will be deleting this page (probably?) and all of the docs will be converted to GJS 👍 So I doubt it will be missed 😂

Reply:
Added info about the RFC stage.

I do see value in these docs after reaching "Recommended". While the format will be adopted in all other pages, we'll need some place to link to explaining what is `.gjs`/ `.gts` and why Ember uses a custom format.

Other frameworks have this too:
- https://react.dev/learn/writing-markup-with-jsx
- https://svelte.dev/docs/svelte-components
- https://vuejs.org/api/sfc-spec